arangod/Pregel/REST/RestControlPregelHandler.cpp

```diff
@@ -171,6 +172,20 @@ void RestControlPregelHandler::handleGetRequest() {
     auto c = _pregel.conductor(executionNumber);
 
     if (nullptr == c) {
+      if (_pregel._statusActors.contains(executionNumber)) {
+        auto statusActor = _pregel._statusActors[executionNumber];
+        auto state =
+            _pregel._actorRuntime->getActorStateByID<pregel::StatusActor>(
+                statusActor.id);
```

Comment:
In an ideal world this would be a function `getStatusActorFor(executionNumber)`; I realise this is only used in this one place and I am nitpicking.